docs/sources/ocimetrics.md

```diff
@@ -0,0 +1,73 @@
+# Oracle Cloud Infrastructure Metrics (OCIMetrics) source for knative eventing
+
+This event source collects metrics data from the [Oracle Cloud](https://cloud.oracle.com).
+
+## Prerequisites
+
+An Oracle Cloud account with permissions to inspect and read metrics for the
+Oracle Cloud Infrastructure (OCI) compartment is required.  In addition, the
+account must have the account API secret added.
+
+For additional information on how to create an API key and associate it with
+your Oracle Cloud user, go to: https://docs.cloud.oracle.com/en-us/iaas/Content/API/Concepts/apisigningkey.htm#Required_Keys_and_OCID
```

Comment:
Maybe use a [Mardown hyperlink](http://rick.roll) instead of a raw URL, it looks nicer in a user facing documentation. I see you did it in the test of the doc.

Reply:
Done!